js/scenes/Surveys/Pages/Surveys.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ScrollView, Text, View } from 'react-native';
+import { GoalsModal, RadioAnswers, WarningModal } from '../SurveyComponents/SurveysComponents';
+import ProgressBar from '../../../components/ProgressBar';
+import LeftNavArrow from '../../../components/LeftNavArrow';
+import RightNavArrow from '../../../components/RightNavArrow';
+import { styles } from '../styles';
+
+const Surveys = ({
+  dispatchAnswer,
+  displayGoalModal,
+  closeWarningModal,
+  goalModalState,
+  warningModalState,
+  question,
+  questions,
+  nextQuestion,
+  previousQuestionFunc,
+  prevQuestion,
+  selectedValue,
+  speakerGoals
+}) => {
+  return (
+    <ScrollView style={styles.sceneContain}>
+      <View style={styles.background}>
+        {(question) ?
+          <View>
+            <GoalsModal
+              onPress={() => displayGoalModal(!goalModalState)}
+              modalState={goalModalState}
+              speakerGoals={speakerGoals}
+            />
+            <WarningModal 
+              onPress={() => closeWarningModal()}
+              modalState={warningModalState}
+            />
+            <Text style={styles.titleQuestion}>
+              {questions.indexOf(question) + 1}. {question.question}
+            </Text>
+            <View style={styles.questionsContainer}>
+              {question.answers.map(answer => (
+                <RadioAnswers 
+                  key={answer.answer}
+                  onPress={() => dispatchAnswer(question.question, question.quality, answer)}
+                  selectedValue={answer.value === selectedValue}
+                  value={answer.value}
+                  answer={answer.answer}
+                  subtext={answer.subtext}
+                />
+              ))}
+            </View>
+          </View>
+          : null}
+        <View style={styles.progressContainer}>
+          {(questions.indexOf(question) === 0)
```

Comment:
If it's a 50/50 logic use ` or = ||`  or `and = &&`

Reply:
Done